test/quantization/fx/test_numeric_suite_fx.py

```diff
@@ -662,6 +661,9 @@ def _op_is_unmatchable(op):
                 # RNNDynamicQuantizeHandler
                 pass
             elif qhandler_cls == qp.DefaultNodeQuantizeHandler:
+                # torch.sum does not have quantized equivalents
+                if base_op == torch.sum:
```

Comment:
For context, did we support torch.sum previously and is that being removed now?

Reply:
no, torch.sum is skipped previously as well, we changed the quantize handler from BinaryOpQuantizeHandler to DefaultNodeQuantizeHandler so the skip code is moved here

Reply:
I can see this getting pretty hard to understand.  With this rule, there seem to be at least three sets of ops:
1. quantization supported by fbgemm/qnnpack, and reference ops
2. quantization supported by reference ops
3. quantization not supported at all

Is there any place where users can find which ops belong to which set?  Would it be possible to move this to a mapping, so the information is easy to understand?  Something like

```
op_to_supported_backends = {
  torch.sum: None,
  torch.add: {BACKEND_FBGEMM, BACKEND_QNNPACK, BACKEND_REFERENCE},
  ...
}
```

Reply:
above fine to be done in a future PR

Reply:
yeah that's true, we don't have this right now, will make this clearer to understand in the future. Basically quantization will be configurable and the configuration will be specific to backend, so we'll not be tracking what is supported by reference ops in the future and we'll only need to track ops supported by some specific backend.